optimizely/optimizely.py

```diff
@@ -422,7 +424,8 @@ def activate(self, experiment_key, user_id, attributes=None):
 
         # Create and dispatch impression event
         self.logger.info('Activating user "%s" in experiment "%s".' % (user_id, experiment.key))
-        self._send_impression_event(project_config, experiment, variation, user_id, attributes)
+        self._send_impression_event(project_config, experiment, variation, experiment.key,
+                                    "experiment", user_id, attributes)
```

Comment:
@pawels-optimizely Should we add `experiment` to the `DecisionSources` enum instead of using hardcoded string?

Reply:
we can, go-sdk is using hardcoded value too, and since it's used only once I don't really care.

Reply:
I think we should add to `DecisionSources` and use the value from there.